pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -59,11 +59,11 @@ type CronWorkflowSpec struct {
 // CronWorkflowStatus is the status of a CronWorkflow
 type CronWorkflowStatus struct {
 	// Active is a list of active workflows stemming from this CronWorkflow
-	Active []v1.ObjectReference `json:"active,omitempty" protobuf:"bytes,1,rep,name=active"`
+	Active []v1.ObjectReference `json:"active" protobuf:"bytes,1,rep,name=active"`
 	// LastScheduleTime is the last time the CronWorkflow was scheduled
-	LastScheduledTime *metav1.Time `json:"lastScheduledTime,omitempty" protobuf:"bytes,2,opt,name=lastScheduledTime"`
+	LastScheduledTime *metav1.Time `json:"lastScheduledTime" protobuf:"bytes,2,opt,name=lastScheduledTime"`
 	// Conditions is a list of conditions the CronWorkflow may have
-	Conditions Conditions `json:"conditions,omitempty" protobuf:"bytes,3,rep,name=conditions"`
+	Conditions Conditions `json:"conditions" protobuf:"bytes,3,rep,name=conditions"`
```

Comment:
These changes are necessary for the JSON merge patch. For example, if we want to clear the `Active` list, JSON Merge Patch needs `{active: []}` to be present in the patch. If `omitempty` is included, it won't be present (since it is empty and therefore omitted) and JSON merge patch won't remove it.